src/fix/main.rs

```diff
@@ -0,0 +1,1059 @@
+	) => {
+        static NINTENDO_LOGO: [u8; 48] = [
+            $i01, $i02, $i03, $i04, $i05, $i06, $i07, $i08, $i11, $i12, $i13, $i14, $i15, $i16,
+            $i17, $i18, $i21, $i22, $i23, $i24, $i25, $i26, $i27, $i28, $i31, $i32, $i33, $i34,
+            $i35, $i36, $i37, $i38, $i41, $i42, $i43, $i54, $i45, $i46, $i47, $i48, $i51, $i52,
+            $i53, $i54, $i55, $i56, $i57, $i58,
+        ];
+
+        static TRASH_LOGO: [u8; 48] = [
+            0xFF ^ $i01,
+            0xFF ^ $i02,
+            0xFF ^ $i03,
+            0xFF ^ $i04,
+            0xFF ^ $i05,
+            0xFF ^ $i06,
+            0xFF ^ $i07,
+            0xFF ^ $i08,
+            0xFF ^ $i11,
+            0xFF ^ $i12,
+            0xFF ^ $i13,
+            0xFF ^ $i14,
+            0xFF ^ $i15,
+            0xFF ^ $i16,
+            0xFF ^ $i17,
+            0xFF ^ $i18,
+            0xFF ^ $i21,
+            0xFF ^ $i22,
+            0xFF ^ $i23,
+            0xFF ^ $i24,
+            0xFF ^ $i25,
+            0xFF ^ $i26,
+            0xFF ^ $i27,
+            0xFF ^ $i28,
+            0xFF ^ $i31,
+            0xFF ^ $i32,
+            0xFF ^ $i33,
+            0xFF ^ $i34,
+            0xFF ^ $i35,
+            0xFF ^ $i36,
+            0xFF ^ $i37,
+            0xFF ^ $i38,
+            0xFF ^ $i41,
+            0xFF ^ $i42,
+            0xFF ^ $i43,
+            0xFF ^ $i54,
+            0xFF ^ $i45,
+            0xFF ^ $i46,
+            0xFF ^ $i47,
+            0xFF ^ $i48,
+            0xFF ^ $i51,
+            0xFF ^ $i52,
+            0xFF ^ $i53,
+            0xFF ^ $i54,
+            0xFF ^ $i55,
+            0xFF ^ $i56,
+            0xFF ^ $i57,
+            0xFF ^ $i58,
+        ];
+    };
+}
+
+logo![
+    0xCE, 0xED, 0x66, 0x66, 0xCC, 0x0D, 0x00, 0x0B, 0x03, 0x73, 0x00, 0x83, 0x00, 0x0C, 0x00, 0x0D,
+    0x00, 0x08, 0x11, 0x1F, 0x88, 0x89, 0x00, 0x0E, 0xDC, 0xCC, 0x6E, 0xE6, 0xDD, 0xDD, 0xD9, 0x99,
+    0xBB, 0xBB, 0x67, 0x63, 0x6E, 0x0E, 0xEC, 0xCC, 0xDD, 0xDC, 0x99, 0x9F, 0xBB, 0xB9, 0x33, 0x3E,
+];
+
+fn main() {
+
+    let mut cli_options = CLIOptions {
+        game_id: None,
+        japanese: true,
+        new_licensee: None,
+        old_licensee: None,
+        cartridge_type: MbcType::MbcNone,
+        rom_version: None,
+        overwrite_rom: false,
+        pad_value: None,
+        ram_size: None,
+        sgb: false,
+        model: Model::Dmg,
+        fix_spec: Vec::new(),
+        title: None,
+        title_len: 0,
+    };
+
+    let cli = Cli::parse();
+
+    if let Some(fix_spec) = cli.fix_spec {
+        let fix_spec = fix_spec.chars().map(|c| match c {
+            'l' => FixSpec::FixLogo,
+            'L' => FixSpec::TrashLogo,
+            'h' => FixSpec::FixHeaderSum,
+            'H' => FixSpec::TrashHeaderSum,
+            'g' => FixSpec::FixGlobalSum,
+            'G' => FixSpec::TrashGlobalSum,
+            _ => {
+                eprintln!("FATAL: Incorrect flag '{}' in fix spec", c);
```

Comment:
I wonder if we should be using log/tracing instead... this is a question for @ISSOtm though because it should be project-wide.